johnsonandjohnson/Bodiless-JS
# fix(test-site): Fix contentful link

## Changes
Change link to the where the guide moved to in documentation.
Also instead of relative link (which works while in edit mode but no static when) to https://johnsonandjohnson.github.io/Bodiless-JS documentation link.

## Test Instructions
* Verify link on home page want to learn more tout that link works in Edit's Review state & Static site.
* Add contentful to some page and verify link works in Edit's Review state & Static site.

## Related Issues
